meinberlin/apps/bplan/serializers.py

```diff
@@ -327,14 +329,13 @@ def _generate_image_filename(self, url_path, file):
 
         root_path, extension = posixpath.splitext(url_path)
         if file:
-            # Workaround: imghdr expects the files position on 0
-            file.seek(0)
-            extension = imghdr.what(file) or "jpeg"
+            file_mime = magic.from_buffer(file.read(), mime=True)
```

Comment:
did you check that we don't need the `file.seek(0)` before the `file.read()` because the reason for that shouldn't have changed I think.